### PR TITLE
Add pipelines

### PR DESCRIPTION
**Pull request by TehreemNisa**

### CODE REVIEW

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull stakateruser/nexus3:SNAPSHOT-PR-3.38.0-keycloak_plugin0.5.0-5`

**Review by usamaahmadkhan — DISMISSED** [review]

File: .github/workflows/pull_request.yml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+          org.opencontainers.image.source=${{ github.event.repository.clone_url }}
+          org.opencontainers.image.created=${{ steps.prep.outputs.created }}
+          org.opencontainers.image.revision=${{ github.sha }}  
+
+    - name: Comment on PR
+      uses: mshick/add-pr-comment@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.STAKATER_GITHUB_TOKEN }}
+      with:
+        message: '@${{ github.actor }} Image is available for testing. `docker pull ${{ secrets.STAKATER_DOCKERHUB_USERNAME }}/nexus3:${{ steps.generate_tag.outputs.GIT_TAG }}`'
```
Comment (usamaahmadkhan) [inline .github/workflows/pull_request.yml]: '@${{ github.actor }} Image is available for testing. `docker pull ${{ env.IMAGE_REPOSITORY }}:${{ steps.generate_tag.outputs.GIT_TAG }}`'

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull stakater/nexus3/nexus3:SNAPSHOT-PR-3.38.0-keycloak_plugin0.5.0-5`

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull stakater/nexus3:SNAPSHOT-PR-3.38.0-keycloak_plugin0.5.0-5`

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull stakater/nexus3:SNAPSHOT-PR-3.38.0-keycloak_plugin0.5.0-5-${sha:0:8}`

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull stakater/nexus3:SNAPSHOT-PR-3.38.0-keycloak_plugin0.5.0-5-${sha:0:4}`

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull stakater/nexus3:SNAPSHOT-PR-3.38.0-keycloak_plugin0.5.0-5-${sha}`

**Comment by stakater-user** [comment]
@TehreemNisa Yikes! You better fix it before anyone else finds out! [Build](https://github.com/nexus3/commit/e8fcbfdfe93e182580dc3f91a0d7b2ed7a3ad052/checks) has Failed!

**Comment by stakater-user** [comment]
@TehreemNisa Yikes! You better fix it before anyone else finds out! [Build](https://github.com/nexus3/commit/c70fd2f5c4ab7948fc467232ccabb1fd2b9c6b2d/checks) has Failed!

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull stakater/nexus3:SNAPSHOT-PR-3.38.0-keycloak_plugin0.5.0-5-0ea8025f`

**Comment by stakater-user** [comment]
@TehreemNisa Yikes! You better fix it before anyone else finds out! [Build](https://github.com/nexus3/commit/1807643e39a83274a079c7879f06f89063ebfb58/checks) has Failed!

**Comment by stakater-user** [comment]
@TehreemNisa Yikes! You better fix it before anyone else finds out! [Build](https://github.com/nexus3/commit/c593631b66f43ad18f1527dddcad08a433d8e003/checks) has Failed!

**Comment by stakater-user** [comment]
@TehreemNisa Yikes! You better fix it before anyone else finds out! [Build](https://github.com/nexus3/commit/691548d8de39b5387c023c99bd93bfd210be2446/checks) has Failed!

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull stakater/nexus3:SNAPSHOT-PR-3.38.0-keycloak_plugin0.5.0-5-8eed5b22`

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull stakater/nexus3:SNAPSHOT-PR-3.38.0-keycloak_plugin0.5.0-5-4b1b99bf`

**Review by usamaahmadkhan — APPROVED** [review]

**Pull request merged**